README.md

```diff
@@ -1,18 +1,32 @@
-# cli
+# Welcome to urfave/cli
 
 [![Run Tests](https://github.com/urfave/cli/actions/workflows/cli.yml/badge.svg)](https://github.com/urfave/cli/actions/workflows/cli.yml)
 [![Go Reference](https://pkg.go.dev/badge/github.com/urfave/cli/v3.svg)](https://pkg.go.dev/github.com/urfave/cli/v3)
 [![Go Report Card](https://goreportcard.com/badge/github.com/urfave/cli/v3)](https://goreportcard.com/report/github.com/urfave/cli/v3)
 [![codecov](https://codecov.io/gh/urfave/cli/branch/main/graph/badge.svg?token=t9YGWLh05g)](https://codecov.io/gh/urfave/cli)
 
-cli is a simple, fast, and fun package for building command line apps in Go. The
-goal is to enable developers to write fast and distributable command line
-applications in an expressive way.
+urfave/cli is a **declarative**, simple, fast, and fun package for building command line tools in Go featuring:
+
+- nestable commands with alias and prefix match support
+- flexible and permissive help system
+- dynamic shell completion for `bash`, `zsh`, `fish`, and `powershell`
+- `man` and markdown format documentation generation
+- input flag types for primitives, slices of primitives, time, duration, and others
+- compound short flag support (`-a` `-b` `-c` :arrow_right: `-abc`)
+- input value sources including:
+    - environment variables
+    - plain text files
+    - [structured file formats supported via the `urfave/cli-altsrc` package](https://github.com/urfave/cli-altsrc)
 
 ## Documentation
 
-More documentation is available in [`./docs`](./docs) or the hosted
-documentation site at <https://cli.urfave.org>.
+More documentation is available in [`./docs`](./docs) or the hosted documentation site published from the latest release
+at <https://cli.urfave.org>.
```

Comment:
What about https://pkg.go.dev/github.com/urfave/cli#section-documentation ?

Reply:
I'm not sure how best to handle the pkg.go.dev docs in relation to the `./docs` tree that's published via mkdocs and the `gh-pages` branch. Do you think mentioning it here is good? More than the link button alone, that is?

Reply:
I would skip that, but if there us a dedicated documentation section, the autogenerated godoc API should be mentioned there. There is no replacement for it in `./docs`.